internal/schema/ize-spec.json

```diff
@@ -3,31 +3,40 @@
     "type": "object",
     "properties": {
         "aws_profile": {
-            "type": "string"
+            "type": "string",
+            "description": "AWS profile"
         },
         "aws_region": {
-            "type": "string"
+            "type": "string",
+            "description": "AWS region"
         },
         "config_file": {
-            "type": "string"
+            "type": "string",
+            "description": "path to config file"
         },
         "env": {
-            "type": "string"
+            "type": "string",
+            "description": "environment"
         },
         "env_dir": {
-            "type": "string"
+            "type": "string",
+            "description": "environment directory"
         },
         "home": {
-            "type": "string"
+            "type": "string",
+            "description": "user home directory"
         },
         "infra_dir": {
```

Comment:
@psihachina let's change this to `ize_dir` here and everywhere else? Should It be a spearate story? LMK.